share/plugins/note_names/notenames.qml

```diff
@@ -97,32 +100,32 @@ MuseScore {
-               case  3: text.text = qsTranslate("accidental", "Double sharp") + text.text; break;
-               case  4: text.text = qsTranslate("accidental", "Double flat") + text.text; break;
-               case  5: text.text = qsTranslate("accidental", "Natural") + text.text; break;
-               case  6: text.text = qsTranslate("accidental", "Flat-slash") + text.text; break;
-               case  7: text.text = qsTranslate("accidental", "Flat-slash2") + text.text; break;
-               case  8: text.text = qsTranslate("accidental", "Mirrored-flat2") + text.text; break;
-               case  9: text.text = qsTranslate("accidental", "Mirrored-flat") + text.text; break;
-               case 10: text.text = qsTranslate("accidental", "Mirrored-flat-slash") + text.text; break;
-               case 11: text.text = qsTranslate("accidental", "Flat-flat-slash") + text.text; break;
-               case 12: text.text = qsTranslate("accidental", "Sharp-slash") + text.text; break;
-               case 13: text.text = qsTranslate("accidental", "Sharp-slash2") + text.text; break;
-               case 14: text.text = qsTranslate("accidental", "Sharp-slash3") + text.text; break;
-               case 15: text.text = qsTranslate("accidental", "Sharp-slash4") + text.text; break;
-               case 16: text.text = qsTranslate("accidental", "Sharp arrow up") + text.text; break;
-               case 17: text.text = qsTranslate("accidental", "Sharp arrow down") + text.text; break;
-               case 18: text.text = qsTranslate("accidental", "Sharp arrow both") + text.text; break;
-               case 19: text.text = qsTranslate("accidental", "Flat arrow up") + text.text; break;
-               case 20: text.text = qsTranslate("accidental", "Flat arrow down") + text.text; break;
-               case 21: text.text = qsTranslate("accidental", "Flat arrow both") + text.text; break;
-               case 22: text.text = qsTranslate("accidental", "Natural arrow down") + text.text; break;
-               case 23: text.text = qsTranslate("accidental", "Natural arrow up") + text.text; break;
-               case 24: text.text = qsTranslate("accidental", "Natural arrow both") + text.text; break;
-               case 25: text.text = qsTranslate("accidental", "Sori") + text.text; break;
-               case 26: text.text = qsTranslate("accidental", "Koron") + text.text; break;
+               case  1: text.text = qsTranslate("engraving/sym", "Sharp") + text.text; break;
+               case  2: text.text = qsTranslate("engraving/sym", "Flat") + text.text; break;
+               case  3: text.text = qsTranslate("engraving/sym", "Double sharp") + text.text; break;
+               case  4: text.text = qsTranslate("engraving/sym", "Double flat") + text.text; break;
+               case  5: text.text = qsTranslate("engraving/sym", "Natural") + text.text; break;
+               case  6: text.text = qsTranslate("engraving/sym", "Flat-slash") + text.text; break;
```

Comment:
It looks like this list is outdated (when compared to accidental.cpp), and not all these names appear in symnames.cpp. But since it's hidden behind an `if (false)` anyway, it's maybe not worth the tourble to fix that... that do you think?

Reply:
Let's do away with them, seems a remnant from MuseScore 1